react/src/components/card/SprkCard.test.js

```diff
@@ -8,109 +7,39 @@ import SprkCard from './SprkCard';
-            href: 'https://sparkdesignsystem.com/',
-            mediaLinkElement: 'a',
-            imgAlt: 'placeholder image',
-            imgSrc: 'https://spark-assets.netlify.app/desktop.jpg',
-            mediaAnalyticsString: 'Card:teaser-link',
-            mediaVariant: 'img',
-          },
-          title: 'Teaser Card Title',
-          titleFirst: false,
-        }}
-      />,
+  it('should not add standout class if isStandout is false', () => {
+    const wrapper = shallow(<SprkCard isStandout={false} />);
+    expect(wrapper.find('.sprk-c-Card').hasClass('sprk-c-Card--standout')).toBe(
+      false,
     );
-    const loadsTeaserComponent =
-      wrapper.find('GetCardContent SprkCardTeaser').length === 1;
-    expect(loadsTeaserComponent).toBe(true);
   });
 
-  it('should apply the correct class if mediaVariant is icon', () => {
-    const wrapper = mount(
-      <SprkCard
-        variant="teaser"
-        teaserConfig={{
-          bodyText: 'Body text of a teaser card.',
-          cta: {
-            ctaAnalytics: 'test',
-            text: 'Learn More',
-            ctaVariant: 'button',
-            buttonVariant: 'tertiary',
-            href: 'https://sparkdesignsystem.com/',
-          },
-          media: {
-            href: 'https://sparkdesignsystem.com/',
-            mediaLinkElement: 'a',
-            imgAlt: 'placeholder image',
-            imgSrc: 'https://spark-assets.netlify.app/desktop.jpg',
-            mediaAnalyticsString: 'Card:teaser-link',
-            mediaVariant: 'icon',
-            iconName: 'bell',
-          },
-          title: 'Teaser Card Title',
-          titleFirst: false,
-        }}
-      />,
-    );
-    expect(
-      wrapper.find('.sprk-c-Card').hasClass('sprk-u-TextAlign--center'),
-    ).toBe(true);
+  it('should assign data-id when idString has a value', () => {
+    const wrapper = mount(<SprkCard idString="321" />);
+    expect(wrapper.find('[data-id="321"]').length).toBe(1);
   });
 
-  // If highlighted header, should load teaser card
-  it(`should load highlighted header
-      if there is highlightedHeaderConfig`, () => {
-    const wrapper = mount(
-      <SprkCard
-        variant="highlightedHeader"
-        highlightedHeaderConfig={{
-          bodyText: `Highlight header card body text.
-            Lorem ipsum dolor sit amet, doctus invenire vix te.
-            Facilisi perpetua an pri, errem commune mea at,
-            mei prima tantas signiferumque at. Numquam.`,
-          title: 'Highlight Header Card Title',
-          description: 'Highlight Header Description',
-        }}
-      />,
-    );
-    const loadsHighlightedHeaderComponent =
-      wrapper.find('GetCardContent SprkCardHighlightedHeader').length === 1;
-    expect(loadsHighlightedHeaderComponent).toBe(true);
+  it('should not assign data-id when idString is not provided', () => {
+    const wrapper = mount(<SprkCard />);
+    expect(wrapper.find('[data-id="321"]').length).toBe(0);
```

Comment:
Update to check to see if they `data-id` attribute exists. Not necessarily this specific value.

Reply:
Good catch, updated!